### PR TITLE
Add Gilded Mechafrog pet (Dragonflight/10.0.7)

### PR DESCRIPTION
**Pull request by cyriun**
forbidden reach map id addition might be a duplicate

### CODE REVIEW

**Comment by cyriun** [comment]
i just click mark as resolved on these right because i think it auto resolves them?

**Comment by Duckwhale** [comment]
> i just click mark as resolved on these right because i think it auto resolves them?

In this case, the change is already merged (as part of the other PR). Your local branch just doesn't have it yet.

You can update the branch by doing a rebase after updating your local `master`:

image

Then you can simply replay the changes from master on top of the branch (there may be conflicts to resolve):

image

In this case, there's two conflicts. Let me know if you want me to sort that out, otherwise I'll let you try the above first :)

File: DB/Pets/Dragonflight.lua [hunk 1]
```diff
@@ -38,6 +38,21 @@ local dragonflightPets = {
...
+	["Gilded Mechafrog"] = {
+		cat = CONSTANTS.ITEM_CATEGORIES.DRAGONFLIGHT,
+		type = CONSTANTS.ITEM_TYPES.PET,
+		method = CONSTANTS.DETECTION_METHODS.USE,
+		name = L["Gilded Mechafrog"],
+		itemId = 204079,
+		items = { 203699, 203700 },
+		spellId = 403072,
+		creatureId = 202484,
+		sourceText = L["Inside Zakera Vault Collect Neltharion Gift Tokens to hand in to Prototype Tinker-Tron to recieve Tattered Gift Package which can drop the Gilded Mechafrog"],
```
Comment (Duckwhale) [inline DB/Pets/Dragonflight.lua:50]: Typo: receive

File: Locales.lua [hunk 2]
```diff
@@ -1883,6 +1883,10 @@ L["This mount can only drop after hitting renown 25 with Dragonscale Expedition.
 L["Expedition Scout's Pack"] = true
 L["Ohuna Companion"] = true
 L["Bakar Companion"] = true
+L["Entrance to Zskera Vault"] = true
+L["Gilded Mechafrog"] = true
+L["Inside Zakera Vault Collect Neltharion Gift Tokens to hand in to Prototype Tinker-Tron to recieve Tattered Gift Package which can drop the Gilded Mechafrog"] =
```
Comment (Duckwhale) [inline Locales.lua:1888]: Typo: receive

File: DB/SharedConstants.lua [hunk 3]
```diff
@@ -113,6 +113,7 @@ C.UIMAPIDS = {
 	OHN_AHRAN_PLAINS = 2023,
 	THE_AZURE_SPAN = 2024,
 	THALDRASZUS = 2025,
+	THE_FORBIDDEN_REACH = 2151,
```
Comment (Duckwhale) [inline DB/SharedConstants.lua:116]: This should go away automatically after a rebase, since the diff from the updated `master` would be empty.

**Review by Duckwhale — CHANGES_REQUESTED** [review]
Should be ready to merge after a rebase, except for the typo I guess 🚀

**Comment by cyriun** [comment]
ok should be good now, i also removed "Bonesifter Marwak" because that was the same case as the duplicate map id

**Comment by Duckwhale** [comment]
> ok should be good now, i also removed "Bonesifter Marwak" because that was the same case as the duplicate map id

It looks like you performed a merge instead of a rebase. That makes it difficult to work with locally (can't rebase, squash or reorder), and it also branches off in the history. Finally, it keeps the original (redundant) changes around and simply undoes them afterwards. In this case it's probably not useful, so if you don't mind we can undo that and do a proper rebase (as pictured in the screenshots)?

See also https://git-scm.com/book/en/v2/Git-Branching-Rebasing and https://www.atlassian.com/git/tutorials/merging-vs-rebasing

**Comment by cyriun** [comment]
yeah im not too sure what happened. So i just revert these two?

chrome_S2CO0ZylDg

**Comment by Duckwhale** [comment]
> yeah im not too sure what happened. So i just revert these two?
> 
> chrome_S2CO0ZylDg

You can "undo" merge commits, then do the rebase workflow outlined above ;) Let me know if it isn't working!

**Comment by cyriun** [comment]
ok i think that should have worked, if it didnt, i dont even know anymore haha

**Comment by Duckwhale** [comment]
> ok i think that should have worked, if it didnt, i dont even know anymore haha

Sorry, my bad. I didn't mean revert (which creates a new commit that undoes the selected changes), but "undo" in GitHub Desktop:

image

This would remove the merge commits entirely, allowing you to do the rebase. Also check the list of commits, it still has the merges.

**Comment by cyriun** [comment]
gotcha and yeah i cannot see undo anywhere so i hit revert thinking it was the same thing haha

**Review by Duckwhale — APPROVED** [review]

**Pull request merged**